src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,49 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, E> {
 
+    private final T firstObject;
+    private final E secondObject;
+
+    public Pair(T firstObject, E secondObject) {
+        this.firstObject = firstObject;
+        this.secondObject = secondObject;
+    }
+
+    public static <T, E> Pair of(T first, E second) {
+        return new Pair(first, second);
+    }
+
+    public T getFirst() {
```

Comment:
```suggestion
    public T getFirstObject() {
```
the same with other getter